presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environments.java

```diff
@@ -47,4 +52,24 @@ public static void pruneEnvironment()
             throw new UncheckedIOException(e);
         }
     }
+
+    public static List<Class<EnvironmentProvider>> findByBasePackage(String packageName)
```

Comment:
`Class<EnvironmentProvider>` type describes `EnvironmentProvider.class` thing
for `class Foo extends EnvironmentProvider`, `Foo.class` can be assigned to `Class<Foo>`, `Class<? extends EnvironmentProvider>`, but not to `Class<EnvironmentProvider>`, so:

```suggestion
    public static List<Class<? extends EnvironmentProvider>> findByBasePackage(String packageName)
```